kahuna/public/js/usage-rights/usage-rights-editor.js

```diff
@@ -77,8 +77,24 @@ usageRightsEditor.controller(
 
     // setting our initial values
     editsApi.getUsageRightsCategories().then(cats => {
-        ctrl.categories = cats;
-        ctrl.originalCats = cats;
+        const copy = angular.copy(cats);
+
+        copy.forEach(cat => {
+            cat.properties.forEach((property, i) => {
```

Comment:
Could we not use a map here and reassign it to a new const instead of the array manipulations below?